wp-js-plugin-starter.php

```diff
@@ -34,4 +34,20 @@ function page_templates_register() {
 function page_templates_enqueue() {
     wp_enqueue_script( 'plugin-sidebar-js' );
 }
-add_action( 'enqueue_block_editor_assets', 'page_templates_enqueue' );
\ No newline at end of file
+add_action( 'enqueue_block_editor_assets', 'page_templates_enqueue' );
+
+/**
+ * Enqueue block styles.
+ */
+function enqueue_editor_styles() {
+	$style_file = is_rtl()
+		? 'index.rtl.css'
+		: 'index.css';
+	wp_enqueue_style(
+		'starter-page-templates-editor-style',
+		plugins_url( 'dist/' . $style_file, __FILE__ ),
+		array(),
+		filemtime( plugin_dir_path( __FILE__ ) . 'dist/' . $style_file )
+	);
+}
+add_action( 'enqueue_block_editor_assets', 'enqueue_editor_styles', 100 );
```

Comment:
Could this be combined with `page_templates_enqueue()`?

Reply:
I was thinking of keeping CSS separate from JS because enqueuing is "noisy" and confusing enough without having to distinguish between CSS and JS enqueues. That said I"m open to changing if required.